### PR TITLE
Feature components

### PR DESCRIPTION
**Pull request by kimothokamau**
- This pull request adds a design of the calculator.
- React components were used

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,22 @@
...
+//     );
+//   }
+// }
+
+const App = () => (
+  <div>
+    <h1> Calculator</h1>
+    <p>Build it now</p>
+  </div>
+);
```
Comment (MahmoudBakr23) [inline src/App.js]: - [x] Please convert it to a class-based component as the project requirements dictate 🙏 😄 It's so important at this step to practice the class-based react components and experience how they work 😉 
- [x] Also, please don't leave any commented-out code in order to keep your code clean and your app optimized 👍

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @kimothokamau 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Descriptive PR headline, summary, and commit messages
- Descriptive README file
- Design followed

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @kimothokamau  🔥 

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

goodjob

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kimothokamau** [comment]
Thank you :)

**Pull request merged**